content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Getting Started
+weight: 10
+---
+## Goals
+
+* Setup development environment
+* Compile and run first Go program
+
+
+## Setup Development Environment
+
+To develop programs with Go you need the following things:
+
+* Go (compiler / tool chain)
+* Editor
+
+
+### Install Go
+
+To install Go follow the official installation instructions under https://go.dev/doc/install. After the installation you can verify if everything works ok by running `go version` in a terminal.
+
+
+### Editor
+
+Of course you need an editor to edit the Go source files. There are no special requirements for the editor so you can use whatever editor you are familiar with.
+However, among Go developers certain editors are more popular than others and hence there is more tooling and plugins available for them. According to the [Go Developer Survey 2020](https://go.dev/blog/survey2020-results) the two most popular editors are:
+
+* VS Code (41%)
+* GoLand / IntelliJ (35%)
+
+For this course it doesn't matter which editor you use. If you are not sure which editor to choose, we recommend you to use VS Code. It is available for free and provides very good support for Go with the official Go extension provided by the Go project.
+
+
+#### VS Code (optional)
+
+If you don't use VS Code you can skip this section. This section will briefly describe how to install VS Code and the recommended extension for Go development:
+
+* Download and Install VS Code from https://code.visualstudio.com/
+* Start VS Code and open the extensions menu on the left side
+* Search for the Go extension and install it
+
+![VS Code Go Extension](vs_code_golang_extension.png)
+
+The first time you open a Go file the Go plugin will show a warning to you that certain tools are missing. Click on "Install All" to install all missing tools. This will take some time.
+
+![VS Code Go Extension Error Message](vs_code_golang_extension_error_message.png)
+
+
+## Hello World
+
+Now we want to build our first Go program which prints `Hello, World!`.
+To create an new Go project we create a new directory and initialize a Go module in it
```

Comment:
Do we need `go mod init` here? Can we introduce it later when we install third party modules?

Reply:
@jyniybinc You can't run `go build` without initializing a module. One could use `go run` but I recommend to only use `go mod init` and `go build` through out the course. This way attendees only have to learn one concept. Also as soon as you start an only have serious project/program you need `go build` anyway. Further `go run` can be confusing at times. For example if you have multiple files in your package it only compiles the files which you pass to `go run`. In the best case, if used wrong, this leads to a compilation error because a certain identifier is undefined in the worst case it leads to unexpected runtime behavior because an `init()` function is not run.

Reply:
:+1: